apps/system/test/marionette/browser_video_test.js

```diff
@@ -76,6 +76,7 @@ marionette('Video', function() {
       return player.isFullScreen();
     });
 
+    client.helper.wait (1000);
```

Comment:
Is there anything else we can wait for instead of an specific time? Looks like a source for future intermitents